README.yaml

```diff
@@ -94,133 +140,99 @@ usage: |2-
       name       = var.name
       stage      = var.stage
       delimiter  = var.delimiter
-      attributes = compact(concat(var.attributes, list("cluster")))
+      attributes = compact(concat(var.attributes, ["cluster"]))
       tags       = var.tags
     }
 
     locals {
-      # The usage of the specific kubernetes.io/cluster/* resource tags below are required
+      # Prior to Kubernetes 1.19, the usage of the specific kubernetes.io/cluster/* resource tags below are required
       # for EKS and Kubernetes to discover and manage networking resources
       # https://www.terraform.io/docs/providers/aws/guides/eks-getting-started.html#base-vpc-networking
-      tags = merge(var.tags, map("kubernetes.io/cluster/${module.label.id}", "shared"))
-
-      # Unfortunately, most_recent (https://github.com/cloudposse/terraform-aws-eks-workers/blob/34a43c25624a6efb3ba5d2770a601d7cb3c0d391/main.tf#L141)
-      # variable does not work as expected, if you are not going to use custom AMI you should
-      # enforce usage of eks_worker_ami_name_filter variable to set the right kubernetes version for EKS workers,
-      # otherwise the first version of Kubernetes supported by AWS (v1.11) for EKS workers will be used, but
-      # EKS control plane will use the version specified by kubernetes_version variable.
-      eks_worker_ami_name_filter = "amazon-eks-node-${var.kubernetes_version}*"
+      tags = { "kubernetes.io/cluster/${module.label.id}" = "shared" }
```

Comment:
@Nuru  this (the tags) needs to be fixed here https://github.com/cloudposse/terraform-aws-eks-cluster/pull/119/files#diff-07f337e38ed1996f30ee4f04357ab23318c254cb606aaf4745d564cbd546722dL17 as well.
Otherwise, people are complaining about using it with Terraform 1.0.2

```
Error: Error in function call
│
│   on main.tf line 82, in locals:
│   82:   tags = merge(local.common_tags, map("kubernetes.io/cluster/${module.label.id}", "shared"))
│     ├────────────────
│     │ module.label.id will be known only after apply
│
│ Call to function "map" failed: the "map" function was deprecated in Terraform v0.12 and is no longer available; use tomap({ ... }) syntax to write a literal map.
```

Reply:
@aknysh The link did not work for me, so I am not sure where else you wanted changed.

Reply:
@Nuru  it's here https://github.com/cloudposse/terraform-aws-eks-cluster/blob/master/examples/complete/main.tf#L17 in the example
Since you updated README, we need to make it consistent with the example itself